### PR TITLE
Add Greek locale

### PR DESCRIPTION
**Pull request by Maximus7474**
Helping out @L1GHTNINGx on [the posted issue](https://github.com/lbphone/lb-phone-locales/issues/113) by creating a PR to check and make sure the file is up to date.

### CODE REVIEW

File: el-gr.json [hunk 1]
```diff
@@ -0,0 +1,1790 @@
...
+            "REMOVE": "Αφαίρεση",
+            "IN_THIS_ROOM": "Σε Αυτό το Δωμάτιο",
+            "WHOS_WATCHING": "Ποιος παρακολουθεί",
+            "FOLLOW_REQUESTS": "Αιτήματα Ακολούθησης",
+            "CONFIRM": "Επιβεβαίωση",
+            "DELETE": "Διαγραφή",
+            "ANDOTHERS": "{username} + {amount} άλλοι",
+            "PRIVATE_ACCOUNT": "Αυτός ο λογαριασμός είναι ιδιωτικός",
+            "PRIVATE_ACCOUNT_DESCRIPTION": "Ακολουθήστε αυτόν τον λογαριασμό για να δείτε τις φωτογραφίες και τα βίντεό του.",
+            "ERROR": "Error",
```
Comment (Maximus7474) [inline el-gr.json:1102]: Requires translation

File: el-gr.json [hunk 2]
```diff
@@ -0,0 +1,1790 @@
...
+            "AIRDROP_DECLINED": "Απορρίφθηκε",
+            "AIRDROP_NO_USERS": "Δεν υπάρχουν τηλέφωνα κοντά",
+            "WALLPAPER_POPUP": {
+                "TITLE": "Χρήση ως Ταπετσαρία",
+                "TEXT": "Είστε σίγουροι ότι θέλετε να ορίσετε αυτή την εικόνα ως ταπετσαρία;",
+                "CANCEL": "Ακύρωση",
+                "PROCEED": "Συνέχεια"
+            },
+            "COPY_POPUP": {
+                "TITLE": "Success",
```
Comment (Maximus7474) [inline el-gr.json:1530]: Requires translation

**Review by Maximus7474 — COMMENTED** [review]
@L1GHTNINGx could you do the missing translations ?

**Comment by L1GHTNINGx** [comment]
Yes, I can do it, little later after work

Στις Κυρ 6 Οκτ 2024 στις 12:10 μμ ο χρήστης Maximus7474 <
***@***.***> έγραψε:

**Pull request closed without merging**